### ISSUE TITLE
0.3.0 does not support hg38 aligned BAM file? 

### ISSUE THREAD

**Issue by nswh** [KCCG/mity#10]
Tested on 0.3.0 0.2.1 0.1.3
Only 0.1.3 passed 
However, 0.1.3 was developed based on hg19 annotation? 
```
$ docker run --rm -it -v $(pwd):/home drmjc/mity:0.3.0 call --prefix NA12878-2403 NA12878-2403.bam
mity version 0.3.0
Calling mitochondrial variants
Running FreeBayes in sensitive mode
FreeBayes failed: b"unable to find FASTA index entry for 'chrM'\n"

$ docker run --rm -it -v $(pwd):/home drmjc/mity:0.2.1 call --prefix NA12878-2403 NA12878-2403.bam
mity version 0.2.1
Calling mitochondrial variants
Traceback (most recent call last):
  File "/usr/local/bin/mity", line 29, in <module>
    args.func(args)
  File "/usr/local/lib/python3.7/site-packages/mitylib/commands.py", line 45, in _cmd_call
    genome = select_reference_genome(args.reference, None)
  File "/usr/local/lib/python3.7/site-packages/mitylib/util.py", line 305, in select_reference_genome
    assert len(res) == 1
AssertionError

$ docker run --rm -it -v $(pwd):/home drmjc/mity:0.1.3 call --prefix NA12878-2403 NA12878-2403.bam
mity version 0.1.3
Calling mitochondrial variants
Running FreeBayes in sensitive mode
```
BAM file was aligned using Broad GATK reference genome bundle https://console.cloud.google.com/storage/browser/genomics-public-data/resources/broad/hg38/v0/

**Comment by drmjc** [comment]
Hi,
Did you try specifying the reference genome?

`docker run --rm -it -v $(pwd):/home drmjc/mity:0.2.1 call --prefix NA12878-2403 --reference hg38 NA12878-2403.bam`

**Comment by trentzz** [comment]
Hi, the issues you've mentioned should be resolved in the new release of Mity 1.1.0. If you encounter any further problems, please feel free to open a new issue. Thanks!
